src/app/[locale]/(pages)/esci/conferma/page.tsx

```diff
@@ -1,12 +1,47 @@
 'use client';
 
 import { useTranslations } from 'next-intl';
-import SessionExistingComp from '@/app/[locale]/_component/sessionExistingComp/sessionExistingComp';
+import { useState, useEffect } from 'react';
+import SessionActiveComp from '@/app/[locale]/_component/sessionActiveComp/sessionActiveComp';
+import { WebProfileApi } from '@/api/webProfileApiClient';
+import NoSessionActiveComp from '@/app/[locale]/_component/noSessionActiveComp/noSessionActiveComp';
+import { storageUserOps } from '@/app/[locale]/_utils/storage';
+import { SessionState } from '@/api/generated/webProfile/SessionState';
 
-const Session = (): React.ReactElement => {
+const LogoutConfirm = (): React.ReactElement => {
+  const [sessionData, setSessionData] = useState<SessionState>();
   const t = useTranslations('ioesco');
+  const userFromStorage = storageUserOps.read();
+  const isL1 = userFromStorage?.spidLevel === process.env.NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L1;
```

Comment:
How can I log with L1 with hub-spid-login to test this?

Reply:
In the first login page in the sidebar on the right click "Esci da IO" button you will be redirected to a new page. Then on the second page click entra con spid.